src/lib/server/summarize.ts

```diff
@@ -1,8 +1,14 @@
 import { buildPrompt } from "$lib/buildPrompt";
 import { generateFromDefaultEndpoint } from "$lib/server/generateFromDefaultEndpoint";
 import { defaultModel } from "$lib/server/models";
+import {
+	LLM_SUMMERIZATION
+} from "$env/static/private";
 
 export async function summarize(prompt: string) {
+	if (!LLM_SUMMERIZATION) {
+		return prompt;
+	}
```

Comment:
Maybe we can slice the prompt so it will not take too much space in the database. 
```suggestion
	if (!LLM_SUMMERIZATION) {
		return prompt.slice(0, 15);
	}
```